CHANGELOG.MD

```diff
@@ -12,6 +12,9 @@ All notable changes to this project will be documented in this file.
 -->
 ## [Unreleased]
 
+### Added
+- Added new 'where' property to allow server side (mapserver) filtering of operational layer
```

Comment:
hoe weet een ontwikkelaar wat er in de where property geplaatst kan worden?

Reply:
Heb de documentatie van operationele laag bijgewerkt in readme.
Hierbij extra informatie gegeven waar deze where wordt gebruikt en link naar arcgis documtatie toegevoegd.